doc/source/metadata/Reference.rst

```diff
@@ -0,0 +1,577 @@
+
+forecast_reference_time
+***********************
+    
+This CF variable represents the nominal data time or start time of a
+model forecast run,
+and has the ``standard_name`` attribute set to ``forecast_reference_time``.
+
+Ideally, this **should no longer** be used for IMPROVER data.
+
+.. warning::
+
+    Use of ``forecast_reference_time`` in IMPROVER is deprecated
+    as it is at best unhelpful and at worst it is confusing,
+    as IMPROVER generates a blend from multiple sources
+    with different start times so there is no unique data time.
+
+forecast_period
+***************
+
+This CF variable represents the interval between
+the ``forecast_reference_time`` and the validity time (``time``)
+and has the ``standard_name`` attribute set to ``forecast_period``.
+
+Ideally, this **should no longer** be used for IMPROVER data.
+
+.. warning::
+
+    Use of ``forecast_period`` in IMPROVER is deprecated
+    as it is at best unhelpful and at worst it is confusing,
+    as IMPROVER generates a blend from multiple sources
+    with different start times so there is no unique data time.
+
+grid_mapping
+************
+
+This CF attribute provides a label pointing to a separate 
+grid mapping variable, which more fully describes the map projection. 
+
+This **must** be present for gridded data,
+as **must** the associated grid mapping variable.
+
+height
+******
+
+This CF scalar coordinate variable is included in some
+cases to fully describe the quantity of interest.
+
+This **should** be included if there is any ambiguity in the interpretation
+of quantity of interest if it is omitted.
+(e.g. an inclusion of ``height`` with a value of ``1.5 m``
+for the representatiion of screen level.) 
+
+history
+*******
+
+Ideally, this netCDF attribute should provide a list of the applications
+that have modified the original data (i.e. an audit trail),
+with recommended practice being to add a date/time stamp
+(in the form ``YYYY-MM-DDThh:mm:ssZ``) and identify the software package.
+However, in practice, this is far from straightforward for IMPROVER
+as it processes a range of model runs,
+so there is no single, sequential processing chain
+from which to generate such an audit trail,
+making it impossible to accurately maintain previous history information.
+
+This is **not currently set** in IMPROVER.
+
+institution
+***********
+
+This CF attribute specifies where the original data was produced.
+
+This **must** be present and **should** take the name of the institution from
+where the data originated if only data from a single model has been processed.
+However, it **should** be set to the institution running the post-processing
+for multi-model blended data.
+
+latitude
+********
+
+This coordinate variable represents one half of the positional
+information for gridded data held on a
+Latitude-Longitude (strictly, equirectangular) projection.
+This is also used for site positions, which are only provided
+in latitude and longitude.
```

Comment:
I believe this is a Met Office convention, not an IMPROVER limitation.

Reply:
I'm not sure that I would say there is a current Met Office convention in a netCDF context anyway (MONET is so out of date) ... or did you mean a more general convention.

I've not made any change here as I'm think the current statement does not really attribute this, so I don't think it's inaccurate, but feel free to suggest a specific change if you think one is required.

Reply:
Ok. We'll leave this as it is.  I was just thinking that spot data could be defined relative to any grid projection and I don't think IMPROVER would object.